buildSrc/src/main/java/Config.kt

```diff
@@ -132,6 +134,7 @@ object Config {
         val awaitility = "org.awaitility:awaitility-kotlin:4.1.1"
         val mockWebserver = "com.squareup.okhttp3:mockwebserver:${Libs.okHttpVersion}"
         val mockWebserver3 = "com.squareup.okhttp3:mockwebserver:3.14.9"
+        val mockWebserver4 = "com.squareup.okhttp3:mockwebserver:4.9.3"
```

Comment:
Since it's a dev dep, can we update `sentry-apollo` to use the v4 as well instead?

Reply:
Sounds good, will do